table/tables/tables.go

```diff
@@ -483,6 +484,14 @@ func (t *Table) RowWithCols(ctx context.Context, h int64, cols []*table.Column)
 	return v, nil
 }
 
+func getTimeZone(ctx context.Context) *time.Location {
```

Comment:
This function is implemented multiple times, can we just implement it as a method of `SessionVars`?